install-conda-dependencies.sh

```diff
@@ -7,4 +7,6 @@ echo "==== Installing Conda packages"
 conda install --yes pocl numpy pyvisfile pyopencl
 
 export OCL_ICD_VENDORS=$MY_CONDA_DIR/etc/OpenCL/vendors/
-echo 'export OCL_ICD_VENDORS=$CONDA_PREFIX/etc/OpenCL/vendors/' >> $HOME/.bashrc
+
+echo "To use the just-installed pocl, add the following line to your ~/.bashrc "
+echo 'export OCL_ICD_VENDORS=$CONDA_PREFIX/etc/OpenCL/vendors/'
```

Comment:
Why is this necessary? `ocl-icd` as installed via conda should default to this.

Reply:
Fixed.